torch/fx/experimental/proxy_tensor.py

```diff
@@ -26,7 +26,7 @@
 from torch import SymInt, SymFloat
 from torch.utils.weak import WeakTensorKeyDictionary
 
-__all__ = ["PythonKeyTracer", "dispatch_trace", "make_fx", "DecompositionInterpreter", "py_sym_types", "get_innermost_proxy_mode"]
+__all__ = ["PythonKeyTracer", "dispatch_trace", "make_fx", "make_fx_allow_non_fake_inputs", "DecompositionInterpreter", "py_sym_types", "get_innermost_proxy_mode"]
```

Comment:
lint?

Reply:
I think the export is still in progress from when I changed it internally 😅